config.js

```diff
@@ -7,7 +7,7 @@ const { ifRequire, falsyProps } = require('./util');
 const custom = ifRequire('./custom');
 
 const db = Object.assign({
-  name: 'tss',
+  name: 'christineshih', // CHANGE BACK TO tss
```

Comment:
It looks like you didn't intend to commit this. Or you could drop the `christineshih` database and let the app create a `tss` one.